Makefile

```diff
@@ -4,6 +4,7 @@ export GO111MODULE := on
 export SHELL := /bin/bash
 
 GOPKG := github.com/veraison/ar4si
+GOPKG += github.com/veraison/ar4si/arc/cmd
```

Comment:
Wouldn't `github.com/veraison/ar4si/cmd/arc` be more in line with golang conventions?

Reply:
The `arc/cmd` package is only there for build & test purposes.  The way users are supposed to interact with that specific software artefact is by installing the binary, and that would be done as:
```sh
go install github.com/veraison/ar4si/arc@latest
```
which doesn't expose `cmd`.